presto-main/src/test/java/com/facebook/presto/sql/planner/assertions/PlanMatchPattern.java

```diff
@@ -152,6 +152,14 @@ public PlanMatchPattern withSymbol(String pattern, String alias)
         return with(new SymbolMatcher(pattern, alias));
     }
 
+    /**
+     * Binds symbol that matches pattern with a given alias. Only one symbol can be bound to a given alias.
+     */
+    public PlanMatchPattern withExactSymbol(String pattern, String alias)
```

Comment:
This matcher matches symbol by pattern as `withSymbol`. The difference is that `withExactSymbol` ensures that there is only one symbol reference for a given pattern. This ensures that the alias relates to exactly same symbol in plan (e.g: not symbol aliased by projection, e.g: `custkey => custkey_6`).